### PR TITLE
Add cloud instructions in k8s infra metrics docs and otel binary usag…

### PR DESCRIPTION
**Pull request by ankit01-oss**
…e docs

### CODE REVIEW

File: docs/tutorial/kubernetes-infra-metrics.md [hunk 1]
```diff
@@ -16,13 +19,120 @@ metrics about Kubernetes to SigNoz.
...
+   helm repo add signoz https://charts.signoz.io
+   ```
+   If the chart is already present, update the chart to the latest using:
+   ```
+   helm repo update
+   ```
+
+   ```bash
+   helm install my-release signoz/k8s-infra  \
+  --set otelCollectorEndpoint=ingest.signoz.io:4317 \
```
Comment (prashant-shahi) [inline docs/tutorial/kubernetes-infra-metrics.md]:
```suggestion
  --set otelCollectorEndpoint=ingest.{region}.signoz.cloud:443 \
```

Best to include the table below after this command:

| Region | Endpoint |
| ------- | --------- |
| US | `ingest.us.signoz.io:443` |
| IN | `ingest.in.signoz.io:443` |
| EU | `ingest.eu.signoz.io:443` |

File: docs/tutorial/kubernetes-infra-metrics.md [hunk 2]
```diff
@@ -16,13 +19,120 @@ metrics about Kubernetes to SigNoz.
...
+	- name: K8S_POD_UID
+	  valueFrom:
+	    fieldRef:
+	      fieldPath: metadata.uid
+  - name: OTEL_EXPORTER_OTLP_INSECURE
+    value: "true"
+  - name: OTEL_EXPORTER_OTLP_ENDPOINT
+    value: $(HOST_IP):4317
+  - name: OTEL_RESOURCE_ATTRIBUTES
+		value: service.name=APPLICATION_NAME,k8s.pod.ip=$(K8S_POD_IP),k8s.pod.uid=$(K8S_POD_UID)
```
Comment (prashant-shahi) [inline docs/tutorial/kubernetes-infra-metrics.md:77]: indentation not right

File: docs/tutorial/kubernetes-infra-metrics.md [hunk 3]
```diff
@@ -16,13 +19,120 @@ metrics about Kubernetes to SigNoz.
...
+  ```
+  
+### Note
+  - Replace `APPLICATION_NAME` with your application name that you wish to see in SigNoz.
+  - In cases of some SDKs, you would have to include `http://` or `https://` prefix for `OTEL_EXPORTER_OTLP_ENDPOINT`
+
+## DataFlow
+  
+  applications ⇒ otel-collectors in your infra ⇒ SigNoz cloud
```
Comment (prashant-shahi) [inline docs/tutorial/kubernetes-infra-metrics.md:86]: not sure if this should be included like this.

Proper pictorial representation would be better than the text with arrows.

File: docs/tutorial/kubernetes-infra-metrics.md [hunk 4]
```diff
@@ -16,13 +19,120 @@ metrics about Kubernetes to SigNoz.
...
+    value: "true"
+  - name: OTEL_EXPORTER_OTLP_ENDPOINT
+    value: $(HOST_IP):4317
+  - name: OTEL_RESOURCE_ATTRIBUTES
+		value: service.name=APPLICATION_NAME,k8s.pod.ip=$(K8S_POD_IP),k8s.pod.uid=$(K8S_POD_UID)
+  ```
+  
+### Note
+  - Replace `APPLICATION_NAME` with your application name that you wish to see in SigNoz.
+  - In cases of some SDKs, you would have to include `http://` or `https://` prefix for `OTEL_EXPORTER_OTLP_ENDPOINT`
```
Comment (prashant-shahi) [inline docs/tutorial/kubernetes-infra-metrics.md]:
```suggestion
  - In cases of some SDKs, you would have to include `https://` prefix for `OTEL_EXPORTER_OTLP_ENDPOINT`
```

File: docs/tutorial/opentelemetry-binary-usage.md [hunk 5]
```diff
@@ -19,7 +22,171 @@ container or deployment which can be easily scaled.
...
+    scrapers:
+      cpu: {}
+      disk: {}
+      load: {}
+      filesystem: {}
+      memory: {}
+      network: {}
+      paging: {}
+      process:
+        mute_process_name_error: true
```
Comment (prashant-shahi) [inline docs/tutorial/opentelemetry-binary-usage.md:63]: Add all the following to supress most of the unwanted errors while scraping process metrics.

```suggestion
        mute_process_name_error: true
        mute_process_exe_error: true
        mute_process_io_error: true
```

File: docs/tutorial/opentelemetry-binary-usage.md [hunk 6]
```diff
@@ -19,7 +22,171 @@ container or deployment which can be easily scaled.
...
+    # Using OTEL_RESOURCE_ATTRIBUTES envvar, env detector adds custom labels.
+    timeout: 2s
+    system:
+      hostname_sources: [os] # alternatively, use [dns,os] for setting FQDN as host.name and os as fallback
+extensions:
+  health_check: {}
+  zpages: {}
+exporters:
+  otlp:
+    endpoint: "ingest.signoz.io:4317"
```
Comment (prashant-shahi) [inline docs/tutorial/opentelemetry-binary-usage.md]:
```suggestion
    endpoint: "ingest.{region}.signoz.io:443"
```

File: docs/tutorial/opentelemetry-binary-usage.md [hunk 7]
```diff
@@ -19,7 +22,171 @@ container or deployment which can be easily scaled.
...
+kill "$(< otel-pid)"
+ ```
+
+## Application Level
+
+Applications in a VM can be instrumented to send telemetry data to the `otel-binary` agent running in same VM.
+
+## DataFlow
+
+Application Instrumentation → Otel-Binary Agent in Same VM → SigNoz Saas
```
Comment (prashant-shahi) [inline docs/tutorial/opentelemetry-binary-usage.md:143]: same as previous comment.

Proper pictorial representation would be better than the text with arrows.

File: docs/tutorial/opentelemetry-binary-usage.md [hunk 8]
```diff
@@ -19,7 +22,171 @@ container or deployment which can be easily scaled.
...
+2. Instrument applications and update Dockerfile(s). Refer to [instrumentation docs](/docs/instrumentation/overview/).
+
+3. Use `OTEL_EXPORTER_OTLP_ENDPOINT` envvar to point to `otel-binary` agent:
+ 
+ ```YAML
+  environment:
+    - OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-binary:4317
+  ...
+  
+ ```
```
Comment (prashant-shahi) [inline docs/tutorial/opentelemetry-binary-usage.md:182]: not sure if this whole section should be part of this page.

Best to break it into multiple small tutorial pages but link them here.

@srikanthccv  what do you think about this?

**Review by prashant-shahi — CHANGES_REQUESTED** [review]
reviewed and left comments

File: docs/tutorial/opentelemetry-binary-usage.md (hunk 5 above)
Comment (prashant-shahi) [reply docs/tutorial/opentelemetry-binary-usage.md:63]: applicable in both Cloud and Self-Host

Comment (ankit01-oss) [reply docs/tutorial/opentelemetry-binary-usage.md:63]: @prashant-shahi for self-host, we don't have any instructions for `config.yaml` file.

File: docs/tutorial/opentelemetry-binary-usage.md (hunk 8 above)
Comment (ankit01-oss) [reply docs/tutorial/opentelemetry-binary-usage.md:182]: ok commented out for now

File: docs/tutorial/kubernetes-infra-metrics.md (hunk 3 above)
Comment (ankit01-oss) [reply docs/tutorial/kubernetes-infra-metrics.md:86]: ok

**Pull request merged**